flink-runtime/src/test/java/org/apache/flink/runtime/highavailability/AbstractHaServicesTest.java

```diff
@@ -44,18 +44,20 @@
 
 import static org.hamcrest.Matchers.contains;
 import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
```

Comment:
We should avoid use hamcrest now. Maybe you can migrate this test class to junit5 and assertj first.

Reply:
Thanks for the comment, I'll migrate this class to junit 5.

Reply:
@reswqa I've migrated the test class to junit5 and assertj, could you take a look at it when you are free? Thanks.